### PR TITLE
Watch later and mobile sidebar

### PR DESCRIPTION
**Pull request by AbhishekkGautam**
### Watch later 
- user can add a video to watch later section. 
- user can delete a particular video from watch later.

### Sidebar ( mobile )
- responsive sidebar 
- user can open the sidebar on the mobile screen by tapping on the hamburger menu.

### CODE REVIEW

File: src/components/MobileSidebar/MobileSidebar.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  opacity: 0;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: var(--modal-bg-100);
+  overflow: hidden;
+  transition: all 0.3s cubic-bezier(0.25, 0.46, 0.43, 0.96);
```
Comment (msdeshmukh009) [inline src/components/MobileSidebar/MobileSidebar.css]: `transition: all;`  is [not recommended](https://www.pno.dev/articles/dont-use-the-all-keyword-in-css-transitions/#:~:text=CSS%20animations%20and%20transitions%20are,when%20done%20the%20wrong%20way.).

**Review by msdeshmukh009 — COMMENTED** [review]
I noticed that if the user is not logged in then it gives an error while doing add to watch later. You can navigate the user to the login page. 
The rest is looking very good.

File: src/components/MobileSidebar/MobileSidebar.css (hunk 1 above)
Comment (AbhishekkGautam) [reply src/components/MobileSidebar/MobileSidebar.css]: done

**Comment by AbhishekkGautam** [comment]
> I noticed that if the user is not logged in then it gives an error while doing add to watch later. You can navigate the user to the login page. The rest is looking very good.

Done. thank you

**Comment by pritam-kr** [comment]
watchlater feature is working as expected, and apart from that 
Everything looks good to me, you can merge it.

**Review by pritam-kr — APPROVED** [review]

**Pull request merged**